src/main/java/com/hibernate/cinema/Main.java

```diff
@@ -45,21 +45,11 @@ public static void main(String[] args) {
                 = (AuthenticationService) injector.getInstance(AuthenticationService.class);
         User andrii = authenticationService
                 .register("andriiromash@gmail.com", "abrakadabra");
-        User bob = authenticationService
-                .register("bobinpop@gmail.com", "bobbystrawbby");
-        try {
-            authenticationService
-                    .register(andrii.getEmail(), "testPassword");
-        } catch (Exception e) {
-            //I know this is not okay to do this.
-            //It was created only to show that everything works fine
-            System.out.println(e);
-        }
-        try {
-            authenticationService.login(andrii.getEmail(), andrii.getPassword());
-            authenticationService.login(bob.getEmail(), "wrongpassword");
-        } catch (AuthenticationException e) {
-            throw new RuntimeException("Login failed: ", e);
-        }
+
+        ShoppingCartService shoppingCartService
+                = (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        shoppingCartService.registerNewShoppingCart(andrii);
```

Comment:
How about registering a new cart at the moment of a user registering?

Reply:
Well, it is no problem to implement it. But I will leave as it is, let's see what mentors think :)